examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exploring the Method of Increments with QEMIST Cloud and Tangelo\n",
+    "\n",
+    "In this notebook, we illustrate how users can leverage QEMIST Cloud and Tangelo to explore the impact of quantum computing on problems tackled with the method of increments (MI) problem decomposition technique, which includes the Frozen Natural Orbital Based Method of Increments (MI-FNO) and incremental Full Configuration Interaction (iFCI). \n",
```

Comment:
In this notebook, we illustrate how users can leverage QEMIST Cloud and Tangelo to explore the impact of quantum computing on problems tackled with the method of increments (MI) problem decomposition technique. We demonstrate workflows using the Frozen Natural Orbital Based Method of Increments (MI-FNO), as well as the incremental Full Configuration Interaction (iFCI)